observability/logging/logging-6.1/log6x-clf-6.1.adoc

```diff
@@ -81,7 +81,7 @@ Outputs are configured in an array under `spec.outputs`. Each output must have a
 
 azureMonitor:: Forwards logs to Azure Monitor.
 cloudwatch:: Forwards logs to AWS CloudWatch.
-elasticsearch:: Forwards logs to an external Elasticsearch instance.
+//elasticsearch:: Forwards logs to an external Elasticsearch instance.
```

Comment:
Why we add '//' here?

Reply:
To comment it out, i've now removed it altogether.

Reply:
added it back